### DISCUSSION TITLE
秀出你的导航站，相互交流学习下！

### DISCUSSION THREAD

**Discussion by 80imike** [hslr-s/sun-panel#101] [General]
我先来，下面是我的导航站，欢迎访问哟！

1. [ IT 人员必备工具箱 ](https://666666.dev/)

![small-so01](https://github.com/hslr-s/sun-panel/assets/1665201/f513afc6-16d2-4a0a-b23d-3f52e6e7afc9)
![small-so02](https://github.com/hslr-s/sun-panel/assets/1665201/c3b50858-8c2f-4538-bdfc-4c853eee7093)
![small-so03](https://github.com/hslr-s/sun-panel/assets/1665201/8aa40afa-5540-43fa-9f0e-fe70d5141d12)

**Comment by dockercore** [comment]
看起来很全啊哈哈哈

**Reply by 80imike** [reply]
必须的 不断更新中 哈哈

**Reply by czxjh17** [reply]
> 必须的 不断更新中 哈哈

你这图标背景是深色的，字体就会变成白色，浅色背景就变成黑色，是用JS控制的吗？

**Comment by xiarof** [comment]
![Uploading screencapture-172-16-1-2-3000-2024-02-24-14_14_09.jpg…]()

**Comment by heihei999** [comment]
哇，这些你这些图标做的都好美观，这个导航面板感觉自动获取图标做的挺差的

**Comment by ym1123** [comment]
https://nav.yumo.cc
这是我的导航页

**Reply by heihei999** [reply]
兄弟，可否把你的配置文件分享一下，我这个好丑啊，还没有动效，我学的不是HTML，这些css啥的搞不太懂。

**Reply by ym1123** [reply]
我导航页的博客里有写配置文件分享,你可以直接访问这个链接来查看我分享的配置思路和源码
https://blog.yumo.cc/archives/sun-panelym-nav-zi-ding-yi-cssjs-yuan-ma-si-lu

**Reply by bing-ke** [reply]
佬教程没了

**Comment by 80imike** [comment]
更新一下

**Reply by leetown3113** [reply]
请问一下怎么修改favicon.ico文件？

**Reply by ym1123** [reply]
在web目录里，修改三个文件，.png .ico .svg

**Reply by KUHE0000** [reply]
修改其中一个就可以？

**Reply by ym1123** [reply]
我这是修改了svg生效的，但是我以防万一就全换了

**Reply by nk-xx** [reply]
> 我这是修改了svg生效的，但是我以防万一就全换了
> 
> 

我挂载了web，里面咋啥文件也没有，没有这几个文件呢？

**Comment by wintsa123** [comment]
![image](https://github.com/hslr-s/sun-panel/assets/47288630/6bf26320-8eb1-4f6c-8fe5-e3e44d2d5a60)
简单添加了ping功能，之后看情况添加监听下载器事件

**Comment by czxjh17** [comment]
看了上面兄弟的，感觉你们都太厉害了！

我喜欢设计师导航的那种样式，最近把我的导航也改成了那种样式。
http://zhounas.com

**Reply by dameini** [reply]
这，对吗？

**Reply by hzc073** [reply]
尼玛，上班呢给我干哪去了。

**Comment by sfeii** [comment]
![捕获](https://github.com/user-attachments/assets/7a6c73e4-c6cd-40db-8789-fb53811212ef)
我的资源导航站，www.sfeii.cn

**Reply by hzc073** [reply]
我很好奇，为什么你的导航站不需要登录？

**Reply by avw95** [reply]
设置公开账户就行

**Comment by breeze303** [comment]
动画效果是怎么实现的？

**Comment by naoki66** [comment]
你们这么多高清logo哪里找的？ 提取网站的分辨率不行

**Comment by xygd555** [comment]
图标的背景颜色是手动配置的吗
